generic/tests/cfg/timedrift.cfg

```diff
@@ -35,21 +38,27 @@
                     drift_threshold = 10
                     drift_threshold_single = 3
                 - with_stop:
-                    # starting from Win2012 Windows no longer relies on number of injected RTC interrupts for time keeping,
-                    # and the time drift is seen because "stop" monitor command also stops PM timer,
-                    # so this is unsupported scenario from Win2012.
-                    Windows:
-                        only Win2000, WinXP, Win2003, WinVista, Win7, Win2008
                     type = timedrift_with_stop
                     stop_interations = 1
                     stop_time = 60
                     drift_threshold = 10
                     drift_threshold_single = 3
                     variants:
                         - clock_host:
+                            # starting from Win2012 Windows no longer relies on number of injected RTC interrupts for time keeping,
+                            # and the time drift is seen because "stop" monitor command also stops PM timer,
+                            # so this is unsupported scenario from Win2012.
+                            Windows:
+                                only Win2000, WinXP, Win2003, WinVista, Win7, Win2008
                             rtc_clock = host
                         - clock_vm:
                             rtc_clock = vm
+                    variants:
+                        - base_utc:
+                            only Linux
+                        - base_localtime:
+                            only Windows
+                        - base_date:                        
```

Comment:
No setting for "rtc_base" for these three variants? 
From my checking result, "rtc_base = localtime" is set in windows.cfg, and "utc" is the default value in qemu_vm.py. Is this what you want for "base_utc" and "base_localtime" ?  and what do you want to set for "base_date" ?